config/default.yml

```diff
@@ -151,46 +172,53 @@ Rails/FindBy:
   Description: 'Prefer find_by over where.first.'
   StyleGuide: 'https://github.com/rubocop-hq/rails-style-guide#find_by'
   Enabled: true
+  VersionAdded: '0.30'
   Include:
     - app/models/**/*.rb
 
 Rails/FindEach:
   Description: 'Prefer all.find_each over all.find.'
   StyleGuide: 'https://github.com/rubocop-hq/rails-style-guide#find-each'
   Enabled: true
+  VersionAdded: '0.30'
   Include:
     - app/models/**/*.rb
 
 Rails/HasAndBelongsToMany:
   Description: 'Prefer has_many :through to has_and_belongs_to_many.'
   StyleGuide: 'https://github.com/rubocop-hq/rails-style-guide#has-many-through'
   Enabled: true
+  VersionAdded: '0.12'
   Include:
     - app/models/**/*.rb
 
 Rails/HasManyOrHasOneDependent:
   Description: 'Define the dependent option to the has_many and has_one associations.'
   StyleGuide: 'https://github.com/rubocop-hq/rails-style-guide#has_many-has_one-dependent-option'
   Enabled: true
+  VersionAdded: '0.50'
   Include:
     - app/models/**/*.rb
 
 Rails/HelperInstanceVariable:
   Description: 'Do not use instance variables in helpers'
   Enabled: true
+  VersionAdded: '2.0'
```

Comment:
📝 This new police exists only in RuboCop Rails. (If we backport to RuboCop core, I think that we will change the version at that time.)

I'd like to try to merge this version `2.0` if there is no problem. WDYT?